tests/vse_sync_pp/parsers/test_phc2sys.py

```diff
@@ -0,0 +1,46 @@
+### SPDX-License-Identifier: GPL-2.0-or-later
+
+"""Test cases for vse_sync_pp.parsers.phc2sys"""
+
+from unittest import TestCase
+from decimal import Decimal
+
+from vse_sync_pp.parsers.phc2sys import (
+    TimeErrorParser,
+)
+
+from .test_parser import ParserTestBuilder
+
+class TestTimeErrorParser(TestCase, metaclass=ParserTestBuilder):
+    """Test cases for vse_sync_pp.parsers.phc2sys.TimeErrorParser"""
+    constructor = TimeErrorParser
+    id_ = 'phc2sys/time-error'
+    elems = ('timestamp', 'terror', 'state', 'delay')
+    accept = (
+        (   'phc2sys[681011.839]: [ptp4l.0.config] '
+            'CLOCK_REALTIME phc offset         8 s2 freq   +6339 delay    502',
+            (Decimal('681011.839'), 8, 's2', 502),
+        ),
+    )
+    reject = ()
+    discard = (
+        'foo bar baz',
+    )
+    file = (
+        '\n'.join((
+            'foo',
+            'phc2sys[847914.839]: [ptp4l.0.config] '
+            'CLOCK_REALTIME phc offset          1 s2 freq      +639 delay    102',
+            'bar',
+            'phc2sys[847915.839]: [ptp4l.0.config] '
+            'CLOCK_REALTIME phc offset          0 s2 freq      +639 delay    102',
+            'baz',
+            'phc2sys[847916.839]: '
```

Comment:
Acknowledging that config file item is absent here (I still propose it needs to be in `accept`).